components/eam/bld/namelist_files/use_cases/2000_scream_lr.xml

```diff
@@ -80,13 +80,13 @@
 <use_hetfrz_classnuc>.false.</use_hetfrz_classnuc>
 <aerodep_flx_type>'CYCLICAL'</aerodep_flx_type>
 <aerodep_flx_datapath>atm/cam/chem/trop_mam/aero</aerodep_flx_datapath>
-<aerodep_flx_file nlev="72">mam4_0.9x1.2_L72_2000clim_c170323.nc</aerodep_flx_file>
```

Comment:
Same comment as above about mixing F2000 and F2010.

Reply:
reverted.